src/main/java/com/zzpj/controllers/PublisherController.java

```diff
@@ -0,0 +1,62 @@
+package com.zzpj.controllers;
+
+import com.zzpj.dtos.PublisherDto;
+import com.zzpj.entities.Publisher;
+import com.zzpj.services.interfaces.PublisherService;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequestMapping("/publisher")
```

Comment:
Przyjęłam konwencję, że ścieżki endpointów w liczbie mnogiej, ale to można przy okazji kiedyś zmienić.